### ISSUE TITLE
do not reset tagged files if there's an error during an operation

### ISSUE THREAD

**Issue by jose1711** [vifm/vifm#432]
consider `/tmp/source` directory containing these files:
`1 2 3 4 5 6 7 8 9`
and `/tmp/destination` containing just a single file named `7`. open the directories in `vifm` side-by-side, tag files 1, 3, 5, 7 and 9 in `/tmp/source` and enter `:copy` command. you'll get an error message (file already exists) and the tagging is gone. it still can be restored using `gs` but it would be great if `vifm` has an option to do this automatically.

**Comment by xaizek** [comment]
Tried this now with `:keepsel copy`. Selection isn't reset, but error message isn't visible either because of "N files selected" message displayed whenever selection isn't empty.

**Comment by jose1711** [comment]
> Selection isn't reset, but error message isn't visible either because of "N files selected" message displayed whenever selection isn't empty.

Yes, that does not help much. With this setup user does not have a clue that in fact there is an issue preventing copying from working.

**Comment by xaizek** [comment]
That should be fixed now. It's wasn't specific to `:copy`, but how statusbar messages were handled (changing it doesn't seem to have caused bugs with messages or I haven't seen them yet).
